docs/manifest_content.md

```diff
@@ -988,6 +988,7 @@ diagnostic | copy | /Packages/Plugins/Microsoft.Powershell.DSC/\*/DSCVersion.xml
 diagnostic | copy | /Packages/Plugins/Microsoft.Powershell.DSC/\*/DSCWork/HotfixInstallInProgress.dsc
 diagnostic | copy | /Packages/Plugins/Microsoft.Powershell.DSC/\*/DSCWork/PreInstallDone.dsc
 diagnostic | copy | /Packages/Plugins/Microsoft.SqlServer.Management.SqlIaaSAgent/\*/PackageDefinition.xm<br>l
+diagnostic | copy | /WindowsAzure/Logs/Plugins/Microsoft.ManagedServices.ApplicationHealthWindows/\*/\*

```

Comment:
i'm not sure we want to grab ALL files, how about just *.log ?  i'm worried it might try and grab the event json files as they are in the same place

Reply:
Event json files are not in this folder. Instead, it is in a sub folder of this folder. My understanding is, the subfolder of `/WindowsAzure/Logs/Plugins/Microsoft.ManagedServices.ApplicationHealthWindows/\*/\*` will not be collected.

`/WindowsAzure/Logs/Plugins/Microsoft.ManagedServices.ApplicationHealthWindows/\*/\*` contains multiple different files:
- log files from apphealth
- current log file of vmwatch
- old vmwatch log files that are zipped.
We should collect all these files

Reply:
Adding something /*/* will collect everything under the Main directory. /WindowsAzure/Logs/Plugins/Microsoft.ManagedServices.ApplicationHealthWindows
If we are not sure what it will have and how big the content it will be then it's batter not to add * instead of add something specific we need.
Also other concern is if this is too big in size then adding as part of "diagnostic" manifest will impact on performance of the operation.

Reply:
the files we want will be in a sub directory that changes with each version (eg `2.0.14`) hence the *

basically there will be a directory with version and then flat under that with a bunch of log files.

could maybe do `*/*.log` ?

Reply:
Sounds good, I updated it to use */*.log